backend/tracim_backend/command/cleanup.py

```diff
@@ -0,0 +1,248 @@
+                for workspace_id in deleted_workspace_ids:
+                    try:
+                        cleanup_lib.delete_workspace_agenda(workspace_id)
+                    except FileNotFoundError:
+                        print(
+                            "Warning: Cannot delete agenda for workspace {}, agenda not found".format(
+                                workspace_id
+                            )
+                        )
+
+            if deleted_user_ids:
+                deleted_user_ids_str = ['"{}"'.format(user_id) for user_id in deleted_user_ids]
+                print("Delete agenda of users {}".format(", ".join(deleted_user_ids_str)))
+                for user_id in deleted_user_ids:
+                    try:
+                        cleanup_lib.delete_user_agenda(user_id)
+                    except FileNotFoundError:
+                        print(
+                            "Warning: Cannot delete agenda for user {}, agenda not found".format(
+                                user_id
+                            )
+                        )
+
+    def _delete_user_database_info(
+        self,
+        user: User,
+        cleanup_lib: CleanupLib,
+        force: bool = False,
+        best_effort: bool = False,
+        anonymised_user_display_name: typing.Optional[str] = None,
+    ):
+        print('Trying to delete user {}: "{}"'.format(user.user_id, user.email))
+        not_owned_workspace_revisions = cleanup_lib.get_user_revisions_on_other_user_workspace(user)
+        print("---------")
+        if not_owned_workspace_revisions:
+            print(
+                '{} revision of user "{}" in other user workspace found, deleting them, can cause inconsistent'
+                " database.".format(len(not_owned_workspace_revisions), user.user_id)
+            )
+        if best_effort and not_owned_workspace_revisions:
+            # INFO - G.M - 2019-12-13 - We can anonymise user
+            print(
+                'Delete most user "{}" data in database but revision outside his workspace and anonymise it.'.format(
+                    user.user_id
+                )
+            )
+            cleanup_lib.delete_user_associated_data(user)
+            deleted_workspace_ids = cleanup_lib.delete_user_owned_workspace(user)
+            deleted_user_id = cleanup_lib.anonymise_user(
+                user, anonymised_user_display_name=anonymised_user_display_name
+            ).user_id
+            print(
+                'user {} anonymised to "{} <{}>".'.format(
+                    user.user_id, user.display_name, user.email
+                )
+            )
+        elif not_owned_workspace_revisions and not best_effort and not force:
+            print(
+                "ERROR: User {} has revisions in other user workspace. Cannot delete it without"
+                " creating inconsistent database. Rollback changes.".format(user.user_id)
+            )
+            exit(3)
```

Comment:
`3` ? if use other than `1` e should document these error codes

Reply:
fixed.